packages/widget/src/widget.ts

```diff
@@ -65,9 +66,13 @@ class SygmaProtocolWidget
 
   @property({ type: Object }) appMetadata?: AppMetadata;
 
+  // ? where is this used?
```

Comment:
on L131, maybe use this instead of new sdkInitialized state? Or delete isLoading haha